src/blog/event-sourcing-vs-crud.md

```diff
@@ -0,0 +1,23 @@
+---
+title: From CRUD to Event Sourcing
+date: 2018-03-09
+author_github: jroper
+tags: event+sourcing crud
+summary: >
+    A side by side comparison of using Event Sourcing vs using CRUD
+---
+
+As the Lagom project has grown in usage, one of the biggest barriers to adoption that we've seen is Lagom's opinionated approach to using event sourcing for persistence. Many developers looking to write reactive microservices come from a CRUD background, often using relational databases to store data.
+
+Lagom doesn't not support CRUD - you can use easily use any persistence technology available on the JVM in Lagom. However, the Lagom persistence APIs do not offer CRUD as a first class option, and the Lagom documentation pushes users in the direction of event sourcing.
```

Comment:
"doesn't not" is not the clearest phrasing. How about "Lagom's service API _can_ be used alongside CRUD-style persistence"?